src/contracts/PreAMMBatcher.sol

```diff
@@ -218,15 +218,15 @@ contract PreAMMBatcher {
                 address(this),
                 sellOrderToken0.sellAmount
             ),
-            "transferFrom for token0 was not succesful"
+            "unsuccessful transferFrom for token0"
```

Comment:
We will also need to change these messages in the contract. Maybe even shorter is to say "Failed transferFrom for token0"

Reply:
I changed it also in the contract